nasa-jpl/explorer-1
# Removing IE-specific scripts, styles and fonts

This PR removes IE-specific scripts, styles and fonts and introduces some breaking changes.

### Breaking changes

The following resources were deleted and need to be removed from your project's JS and SCSS imports
- `src/js/_detect-ie.js`, including the node polyfill for `global`
- `src/scss/components/_RoboticsDetailFacts.scss`
- `src/scss/components/_SearchResultCard.scss`
- all `.woff` files. This has been accounted for in explorer-1's styles, but be aware of this in case you are importing `.woff` files a la carte.

### To test

1. `npm run dev` for a quick check that nothing has broken in non-IE browsers
2. `npm run storybook` to check storybook stories
